code/game/objects/items/weapons/tools.dm

```diff
@@ -580,6 +605,19 @@
 /obj/item/weapon/weldingtool/mini/flamethrower_screwdriver()
 	return
 
+/obj/item/weapon/weldingtool/abductor
+	name = "alien welding tool"
+	desc = "An alien welding tool. Whatever fuel it uses, it never runs out."
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "welder"
+	toolspeed = 0.1

```

Comment:
i wanted them to be better than the 0.25 two-in-one roundstart CE tools, but i'll scale them back if you want